test/e2e/nstemplatetier_test.go

```diff
@@ -0,0 +1,44 @@
+package e2e
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-e2e/doubles"
+	"github.com/codeready-toolchain/toolchain-e2e/wait"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	client "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestCreateNSTemplateTierAtStartup(t *testing.T) {
+	// given:
+	// given
```

Comment:
two `given`s

Reply:
is it too much? ;)
removed duplicate comment in a788e17